lib/compat/wordpress-6.5/block-bindings/index.php

```diff
@@ -0,0 +1,11 @@
+<?php
```

Comment:
In Gutenberg, usually, there is no `index.php`  file for file loading. Instead, all files are in `/Users/carlosbravo/repos/gutenberg/lib/load.php`

Reply:
Fixed in [fba69b0](https://github.com/WordPress/gutenberg/pull/58089/commits/fba69b0fc4c255d5bf0b7e60c2d9bca182198dc1) and [23e8a73](https://github.com/WordPress/gutenberg/pull/58089/commits/23e8a736668046f57c64b3fb8f374cccf31476a5)